python/pyspark/sql/pandas/utils.py

```diff
@@ -26,11 +26,13 @@ def require_minimum_pandas_version():
     try:
         import pandas
         have_pandas = True
-    except ImportError:
+    except ImportError as error:
         have_pandas = False
+        raised_error = error
     if not have_pandas:
         raise ImportError("Pandas >= %s must be installed; however, "
-                          "it was not found." % minimum_pandas_version)
+                          "it was not found or an error was thrown by pandas"
```

Comment:
It seems that we need an addition space at the end of this string. `pandas"` -> `pandas "`.

Reply:
Addressed in https://github.com/apache/spark/pull/31902/commits/30542f345f739a63c67c876b4b87f7a0240f228e